### PR TITLE
MINIFICPP-1203 - Linter reported redundand blank lines

### PR DESCRIPTION
**Pull request by hunyadi-dev**
Removed linter reported redundant blank lines at the beginning/end of code blocks when cpplint is enabled on the whole project.

Turn off ignore-whitespace to review the PR.

### CODE REVIEW

File: extensions/rocksdb-repos/DatabaseContentRepository.h [hunk 1]
```diff
@@ -64,7 +64,6 @@ class StringAppender : public rocksdb::AssociativeMergeOperator {
   }
 
  private:
-
```
Comment (fgerlits) [inline extensions/rocksdb-repos/DatabaseContentRepository.h:67]: the previous two lines could be removed, too

File: extensions/windows-event-log/wel/WindowsEventLog.cpp [hunk 2]
```diff
@@ -105,7 +105,6 @@ void WindowsEventLogMetadataImpl::renderMetadata() {
       default:
         event_type_index_ = 0;
     };
```
Comment (fgerlits) [inline extensions/windows-event-log/wel/WindowsEventLog.cpp:107]: not related to blank lines, but this `;` is not needed here and will cause a linter error at some point

File: extensions/mqtt/protocol/MQTTC2Protocol.h [hunk 3]
```diff
@@ -86,7 +86,6 @@ class MQTTC2Protocol : public C2Protocol {
   // mqtt controller serviec name.
   std::string controller_service_name_;
 
-
```
[the remark this answers is not in the document]
Comment (hunyadi-dev) [reply extensions/mqtt/protocol/MQTTC2Protocol.h:89]: Probably true, the linter originally only reported this single blank line. I would say it should be another iteration on correcting linter errors until this is removed.

File: extensions/rocksdb-repos/DatabaseContentRepository.h (hunk 1 above)
Comment (hunyadi-dev) [reply extensions/rocksdb-repos/DatabaseContentRepository.h:67]: That is a different error of a different scope than the errors this PR meant to fix.

File: extensions/windows-event-log/wel/WindowsEventLog.cpp (hunk 2 above)
Comment (hunyadi-dev) [reply extensions/windows-event-log/wel/WindowsEventLog.cpp:107]: That is a different error of a different scope than the errors this PR meant to fix.

**Review by martinzink — APPROVED** [review]

**Review by fgerlits — APPROVED** [review]

**Review by szaszm — APPROVED** [review]

**Review by lordgamez — APPROVED** [review]

**Pull request closed without merging**